proteus/mprans/MCorr.py

```diff
@@ -656,6 +666,59 @@ def getResidual(self,u,r):
             self.mesh.exteriorElementBoundariesArray,
             self.mesh.elementBoundaryElementsArray,
             self.mesh.elementBoundaryLocalElementBoundariesArray)
+        elif self.coefficients.mass_correction_reference==1:
```

Comment:
same as above, don't duplicate the call, just pass in the flag, mass_correciton_reference